config/manifests/bases/group-sync-operator.clusterserviceversion.yaml

```diff
@@ -590,7 +590,7 @@ spec:
         path: providers[0].okta.extractLoginUsername
```

Comment:
Can the updated `groupLimit` property in the README be updated in the `description` section of this file?